src/mui-setup.js

```diff
@@ -19,14 +18,15 @@ export default function muiSetup(direction) {
       boxShadow: '0 0 0 2px #3f8ab3 inset',
       outline: 'none',
     };
+    muiConfig.overrides.MuiToolbar = {
+      regular: {
+        minHeight: '49px !important',
+        paddingRight: '24px !important;',
```

Comment:
I do not know why the custom theme is overridden by other themes. I look into it when I work on the v5 theme which is my next PR.